Deploy-CAM.ps1

```diff
@@ -1114,7 +1114,7 @@ function Generate-Certificate-And-Passwords() {
     $rwLocalAdminPassword = ConvertTo-SecureString $rwLocalAdminPasswordStr -AsPlainText -Force
     $CAMConfig.parameters.remoteWorkstationLocalAdminPassword.value = $rwLocalAdminPassword
 
-    Write-Host "Creating Local Admin Password for Cloud Access connector servers"
+    Write-Host "Creating Local Admin Password for Cloud Access Connector servers"
```

Comment:
This was from Paul today. 
FYI: for external documentation, we should use "Cloud Access connector" (note connector starts with lower case). Thanks.